.github/workflows/run_benchmarks.yml

```diff
@@ -0,0 +1,109 @@
+name: "Launching benchmarks"
+
+on: 
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: 'Log level'     
+        required: true
+        default: 'warning'
+      tags:
+        description: 'Test scenario tags'  
+
+#on: push
+
+jobs:
+  deploy:
+    name: Performance of benchmarks
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Running benchmarks for flask application in monolith version
+        uses: appleboy/ssh-action@master
+        with:
+          key: ${{ secrets.EC2_SSH_KEY }}
+          host: ${{ secrets.HOST_DNS }}
+          username: ubuntu
+          port: 22 
+          script:  |
+            docker rm -f $(docker ps -aq); docker system prune --all -f &&\
+             docker-compose -f docker-compose-flask-monolith.yml up --build -d &&\
+              echo Here will be the launch of the tests 
+               echo Here will be the Data Collection 
+                docker rm -f $(docker ps -aq) && docker system prune --all -f 
+                 echo Benchmark for monolith version of flask application completed, system cleared
```

Comment:
I would prefer these commands to be saved in a separate shell file. eg. `infrastructure/run_benchmarks.sh`
This will make it easier for us to manage our processes.

Reply:
We will work on it in next pr